qa-pipelines/tasks/run-test.sh

```diff
@@ -1,16 +1,7 @@
 #!/bin/bash
 set -o errexit
 
-if   [[ $ENABLE_CF_SMOKE_TESTS_PRE_UPGRADE == true ]] || \
-     [[ $ENABLE_CF_SMOKE_TESTS == true ]]; then
-    TEST_NAME=smoke-tests
-elif [[ $ENABLE_CF_BRAIN_TESTS_PRE_UPGRADE == true ]] || \
-     [[ $ENABLE_CF_BRAIN_TESTS == true ]]; then
-    TEST_NAME=acceptance-tests-brain
-elif [[ $ENABLE_CF_ACCEPTANCE_TESTS == true ]] || \
-     [[ $ENABLE_CF_ACCEPTANCE_TESTS_PRE_UPGRADE == true ]]; then
-    TEST_NAME=acceptance-tests
-else
+if [[ -z "${TEST_NAME:-}" ]] ; then
```

Comment:
I love that we can delete the whole above mess now. Is this section strictly necessary though?